spacy_transformers/data_classes.py

```diff
@@ -99,16 +99,24 @@ def from_batch_encoding(cls, token_data: BatchEncoding) -> "WordpieceBatch":
             for tokens in token_data["input_texts"]
         ]
 
+        # The following tensors are intentionally allocated on the CPU to reduce
+        # host-to-device copies.
         numpy_ops = NumpyOps()
+        input_ids = token_data["input_ids"]
+        input_ids_dtype = input_ids.dtype if hasattr(input_ids, "dtype") else None
```

Comment:
Is there any chance that this doesn't have `.dtype`?

Reply:
Yeah, we pass lists in some of the tests.

Reply:
Is it just in the `Dummy` test stuff?

Reply:
Don't think so/not sure - the [failing tests](https://dev.azure.com/explosion-ai/Public/_build/results?buildId=23801&view=logs&jobId=a25e3be4-cea9-5bf9-7ae0-24f1fc94349b&j=a25e3be4-cea9-5bf9-7ae0-24f1fc94349b&t=98bd6853-46be-5a37-e0f4-c5ea95fba4d8) were in `test_pipeline_component.py`.

Reply:
I'm pretty sure in practice it's always something with `.dtype` and we could update `DummyTokenizer`.

Reply:
Ok, I'll do that instead.